contracts/libraries/Pool.sol

```diff
@@ -17,6 +17,12 @@ library Pool {
     using Position for mapping(bytes32 => Position.Info);
     using Position for Position.Info;
 
+    uint16 private constant FEE_MAX = 2 ** 12 - 1;
```

Comment:
total nit (non-blocker): wondering if hex format is more intuitive

```
uint16 private constant FEE_MAX = 0xfff;  // max 12 bits
```

Reply:
oh and technically this is a` FEE_DENOMINATOR_MAX` I do think the denominator aspect is important to clarify.